package.json

```
@@ -7,7 +7,7 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "export": "next build && next export",
+    "export": "next build && next export -o public",
```

If you want to use this to export for now, it should be this (notice the `/` before public) :
```suggestion
    "export": "next build && next export -o /public",
```

But it's probably better to keep it like this :
```suggestion
    "export": "next export",
```

And then, in the dockerfile, use yarn export -o /public